styles.css

```diff
@@ -7,10 +7,178 @@ body {
   overflow-x: hidden;
 }
 
+.background {
+  overflow-x: hidden;
+}
+
 a {
   text-decoration: none;
 }
 
+.error {
+  background-color: white;
+  color: red;
+}
+
+.blur {
+  filter: blur(8px);
+}
+
+.introbox {
+  overflow: hidden;
+}
+
+header {
+  background-color: white;
+  box-sizing: border-box;
+  position: fixed;
+  width: 100%;
+  z-index: 4;
+}
+
+.navbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.nav-menu {
+  display: flex;
+  justify-self: end;
+  align-items: center;
+  gap: 20px;
+}
+
+.nav-brand {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.nav-link {
+  color: #344563;
+  transition: 0.7s ease;
+}
+
+.nav-link:hover {
+  color: #6070ff;
+}
+
+.hamburguer {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+.popup .box2 {
+  top: 10px;
+  height: 100%;
+  width: 100%;
+}
```

Comment:
- [ ] it would be better to use the right close button icon in order to match the Figma design 
- Yours 
<img width="67" alt="Screenshot 2023-04-11 at 22 14 13" src="https://user-images.githubusercontent.com/61790587/231300967-36feb2ab-030e-4da5-af0c-ac2b25b2b82c.png">
- Design 
<img width="569" alt="Screenshot 2023-04-11 at 22 12 41" src="https://user-images.githubusercontent.com/61790587/231300995-634ed9c4-ce1b-4c1a-b1c5-e18d93272a33.png">

Reply:
requirements says as much as possible, should i change what? the color? @youmari

Reply:
@gersonhiggins it would be better to export the right icon from Figma and use it